src/style.css

```diff
@@ -1,3 +1,69 @@
-body {
+:root {
+  --active-link-color: #00ACDC;
+}
+
+html {
+  font-family: Roboto, "Times New Roman", sans-serif;
+}
+
+ body {
+  margin: 0;
+  padding: 0;
+  background-color: rgb(229, 229, 229);
+  font-size: 12px;
+ }
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  padding: 0 50px;
+}
+
+img {
+  margin: 10px 0;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
   margin: 0;
 }
+
+.nav-item {
+  position: relative;
```

Comment:
```suggestion

```

Reply:
but if I remove position: relative - this blue line for "is-active" element disappears.. How can I fix it then?

Reply:
> 
but if I remove position: relative - this blue line for "is-active" element disappears.. How can I fix it then?